inference-engine/src/plugin_api/ie_system_conf.h

```diff
@@ -52,6 +52,17 @@ INFERENCE_ENGINE_API_CPP(std::vector<int>) getAvailableCoresTypes();
  */
 INFERENCE_ENGINE_API_CPP(int) getNumberOfCPUCores(bool bigCoresOnly = false);
 
+/**
+ * @brief      Returns number of CPU logical cores on Linux/Windows (which is considered to be more performance
+ * friendly for servers) (on other OSes it simply relies on the original parallel API of choice, which usually uses the
```

Comment:
please remove the description of "which is considered....~~".

Reply:
Please review the description again. thanks.